tabular/src/autogluon/tabular/predictor/predictor.py

```diff
@@ -480,6 +481,10 @@ def fit(self,
             If 'infer', will automatically construct a FeatureMetadata object based on the properties of `train_data`.
             In this case, `train_data` is input into :meth:`autogluon.tabular.FeatureMetadata.from_df` to infer `feature_metadata`.
             If 'infer' incorrectly assumes the dtypes of features, consider explicitly specifying `feature_metadata`.
+        calibrate: bool, default = False
```

Comment:
Move to kwargs doc section

Reply:
what is "kwargs doc section?" Do you mean docs under `_validate_fit_kwargs`

Reply:
moved